[2024-06-27 12:19:39,917][flwr][INFO] - Starting Flower simulation, config: ServerConfig(num_rounds=3, round_timeout=None)
[2024-06-27 12:19:43,239][flwr][INFO] - Flower VCE: Ray initialized with resources: {'object_store_memory': 3624748646.0, 'memory': 7249497294.0, 'CPU': 16.0, 'node:172.16.0.2': 1.0}
[2024-06-27 12:19:43,240][flwr][INFO] - Initializing global parameters
[2024-06-27 12:19:43,240][flwr][INFO] - Requesting initial parameters from one random client
[2024-06-27 12:19:44,292][flwr][ERROR] - [36mray::launch_and_get_parameters()[39m (pid=11651, ip=172.16.0.2)
  File "/home/ishmam/anaconda3/envs/fl/lib/python3.8/site-packages/flwr/simulation/ray_transport/ray_client_proxy.py", line 135, in launch_and_get_parameters
    client: Client = _create_client(client_fn, cid)
  File "/home/ishmam/anaconda3/envs/fl/lib/python3.8/site-packages/flwr/simulation/ray_transport/ray_client_proxy.py", line 168, in _create_client
    client_like: ClientLike = client_fn(cid)
  File "/home/ishmam/Documents/Research/Federated Learning/TestFL/client.py", line 59, in client_function
    return FlowerClient(train_loaders[int(client_id)],
AttributeError: 'FlowerClient' object has no attribute 'to_client'
